# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/actors/moz.build
# Commit: 2d0e5336e455
# Full Hash: 2d0e5336e455f560cdd345afa57f21b8cdbfc1fb
# Author: Andreas Farre <farre@mozilla.com>
# Date: 2019-08-26 22:30:36
# Regressor Bug: 1563619
# File Overlap Count: 1
# Description:
#   Bug 1563619 - Handle subframe crashes in BrowserParent::ActorDestroy. r=mconley,kmag,peterv
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D40937
# ==============================================================================

diff -r fd3185644de9 -r 2d0e5336e455 browser/actors/moz.build
--- a/browser/actors/moz.build	Mon Aug 26 13:07:26 2019 +0000
+++ b/browser/actors/moz.build	Mon Aug 26 13:08:32 2019 +0000
@@ -46,8 +46,6 @@
     'PromptParent.jsm',
     'RFPHelperChild.jsm',
     'SearchTelemetryChild.jsm',
-    'SubframeCrashChild.jsm',
-    'SubframeCrashParent.jsm',
     'SwitchDocumentDirectionChild.jsm',
     'URIFixupChild.jsm',
     'WebRTCChild.jsm',